app/Http/Middleware/HasRoleInService.php

```diff
@@ -10,17 +10,17 @@
 use Illuminate\Auth\AuthenticationException;
 use Illuminate\Http\Request;
 
-class HasRoleInService
-{
+class HasRoleInService {
```

Comment:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#4-classes-properties-and-methods